packages/gravity-ui-web/src/sass/05-components/03-organisms/00-page-structure/_01-page-footer.scss

```diff
@@ -1,22 +1,21 @@
-$grav-page-breakpoint-medium: gravy-breakpoint(medium);
```

Comment:
You've removed this variable, but then it's declared on line 18 anyway. Maybe we could remove that declaration too, or use it if it seems appropriate.
```
.grav-c-page-footer__lists {
  $grav-page-breakpoint-medium: grav-breakpoint(medium);
```

Reply:
Yeah, I removed the top ones since that's a global variable and probably ought not to be there. I left the second one because it's within the .grav-c-page-footer__lists so it's scoped to that block. Also it's being used twice so that seemed like an reasonable use of a SASS var.

In hindsight though, it might be neater to just use grav-breakpoint(medium) throughout.

Happy to make that change if you agree.

Reply:
for me, i'd prefer the sleek looks of `grav-breakpoint(medium)`

Reply:
Your wish is my command